version.h.in

```diff
@@ -19,6 +19,8 @@
 #ifndef VERSION_H
 #define VERSION_H
 
+#include <cstdint>
```

Comment:
@FlexW Just wondering. Why is this needed?

Reply:
Ah it's not needed anymore. I originally used uint32_t instead of int. But I then decided to stick with int. I remove it :)